tests/test_embeddings_fk_migration.py

```
@@ -0,0 +1,250 @@
+        "INSERT INTO memory_embeddings (memory_id, embedding_json) VALUES (?, ?)",
+        ("test-embedding-1", "[0.1, 0.2, 0.3]"),
+    )
+    conn.execute(
+        "INSERT INTO memory_embeddings (memory_id, embedding_json) VALUES (?, ?)",
+        ("test-embedding-2", "[0.4, 0.5, 0.6]"),
+    )
+    conn.commit()
+    conn.close()
+
+
+def _get_schema(conn, table):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_get_schema`

(ANN202)